index_2.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My childhood pets</title>
+    <link rel="stylesheet" type="text/css" href="style_2.css" />
+  </head>
+  <body>
+    <h1 class="box-header">My childhood pets</h1>
+   <section class="pets">
+    <div class="flex-child jason">Jason  <img class="pet-image" src="https://images.pexels.com/photos/544269/pexels-photo-544269.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1" alt="Jason" /></div>
+   <div class="flex-child paco">Paco <img  src="https://images.pexels.com/photos/162149/dog-black-labrador-black-dog-162149.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1" alt ="Paco" /></div>
+    <div class="flex-child garfield">Garfield <img  src="https://images.pexels.com/photos/6982643/pexels-photo-6982643.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1" alt="Garfield" /></div>
+    <div class="flex-child tiger">Tiger <img  src="https://images.pexels.com/photos/617278/pexels-photo-617278.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1" alt="Tiger" /></div>
+  <div class="flex-child max">Max <img src="https://images.pexels.com/photos/104373/pexels-photo-104373.jpeg?cs=srgb&dl=pexels-mike-b-104373.jpg&fm=jpg&_gl=1*g3sdv7*_ga*NjE4MzQxMDQ4LjE2NjY0MjMyNDY.*_ga_8JE65Q40S6*MTY2NjQyMzI0Ni4xLjEuMTY2NjQyNDczOC4wLjAuMA.." alt="Max" /> </div>
+  <div class="flex-child moritz">Moritz <img src="https://images.pexels.com/photos/372166/pexels-photo-372166.jpeg?cs=srgb&dl=pexels-pixabay-372166.jpg&fm=jpg&_gl=1*14yy15f*_ga*NjE4MzQxMDQ4LjE2NjY0MjMyNDY.*_ga_8JE65Q40S6*MTY2NjQyMzI0Ni4xLjEuMTY2NjQyMzMxOS4wLjAuMA.." alt="Moritz" /> </div></section>
```

Comment:
This looks a bit messy. Remember to always use correct indentation so that the opening tag and the closing tags end up at the same indentation level. Also - always indent children. Compare this snippet with indentation, it gets much easier to read and nice to look at, right? 😇

```
<section class="pets">
  <div class="flex-child jason">
    <p>Jason</p> <!--I added a paragraph tag here-->
    <img
      class="pet-image"
      src="https://images.pexels.com/photos/544269/pexels-photo-544269.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1"
      alt="Jason" />
  </div>
  <div class="flex-child paco">
    Paco
    <img
      src="https://images.pexels.com/photos/162149/dog-black-labrador-black-dog-162149.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1"
      alt ="Paco" />
  </div>
   <div class="flex-child garfield">
     Garfield
     <img
       src="https://images.pexels.com/photos/6982643/pexels-photo-6982643.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1"
       alt="Garfield" />
   </div>
</section>


```